libs/langgraph/langgraph/types.py

```diff
@@ -245,7 +245,20 @@ def __eq__(self, value: object) -> bool:
 
 @dataclasses.dataclass(**_DC_KWARGS)
 class Command(Generic[N]):
-    """One or more commands to update the graph's state and send messages to nodes."""
+    """One or more commands to update the graph's state and send messages to nodes.
+
+    Args:
+        graph: graph to send the command to. Supported values are:
+            - None: the current graph (default)
+            - GraphCommand.PARENT: closest parent graph
+        update: state update to apply to the graph's state at the current superstep.
+        resume: value to resume execution with. Will be used when `interrupt()` is called.
```

Comment:
```suggestion
        resume: value to resume execution with. To be used together with `interrupt()`.
```
Can we link to interrupt() here?